src/components/Courses/CourseCard/CourseCard.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import styles from './CourseCard.module.scss';
+import Button from '../../../common/Button/Button';
+import { ButtonsName } from '../../../assets/text/buttonsName';
+
+interface CourseCardProps {
+	id: string;
+	title: string;
+	description: string;
+	creationDate: string;
+	duration: number;
+	authors: string[];
+}
+
+export const CourseCard: React.FC<CourseCardProps> = (props) => {

```

Comment:
You can return implicitly:
`
 export const CourseCard: React.FC<CourseCardProps> = (props) => ()
`